docs/apm/traces/view-and-investigate-traces.md

```diff
@@ -59,23 +59,66 @@ Queries above 7 days may be slower to load.
+The Trace Query Visualizations allow you to visualize the breakdown of where time was spent in traces for various services in your application, the complexity of your traces as well as errors. By aggregating traces, you can gain insights into anomalies and unexpected behaviors, leading to faster Time-to-Resolution (TTR) and a higher Return on Investment (ROI). There are three types of metrics that Trace Query Visualizations offer:
+* **Trace duration**
+* **Error count**
+* **Span count**
 
-* Quickly understand intermittent duration spikes or slowdowns.
-* Immediately spot the offending service by comparing CPC contribution by service.
+<img src={useBaseUrl('/img/traces/trace-query-view.png')} alt="trace-query-view.png" width="950px" height="500" />
 
-![breakdown](/img/traces/breakdown2.png)
+These metrics can be viewed as either a **timeseries** chart or a **histogram** chart.
+* **Timeseries**. It shows the average trace duration for each time bucket.
+* **Histogram**. It shows the distribution of traces (their count) in different buckets such as duration, number of spans, or errors.
 
-For best results, filter your traces to represent similar traces (traces of same transaction like login). Running this chart for different transaction types will not provide the insights you want. For the same reason, running the chart for all data without any filters is disabled.
+For both charts, you can switch between linear and logarithmic scales to better visualize the data. Note that logarithmic charts will not show any visible bars for buckets with only one trace.
 
-* The height of the bar represents the average trace duration for each time bucket.
-* Each segment represents a Critical Path Contribution of each service from each trace. Services not present in certain traces do not contribute to the value.
-* Click on any of the color segments to focus on this service and drill down to selected timeframe.
-* Click and drag on the chart to zoom in.
+#### Understanding each chart type in detail
 
+* **Trace duration**.
+  * **Timeseries**. It allows you to quickly identify services that are causing spikes or slowdowns in trace duration. It helps you understand the average trace duration for every time bucket, as well as the amount of time each service contributed to the end-to-end duration.
+    - The height of the bar represents the average trace duration for each time bucket.
+    - Each segment represents a Critical Path Contribution of each service from each trace. Services not present in certain traces do not contribute to the value.
+  * **Histogram**. It helps you understand the distribution of trace durations. For example you can notice groups of durations responsible for longer traces during slowdown periods.
+ <img src={useBaseUrl('/img/traces/trace-duration-histogram.png')} alt="trace-duration-histogram.png" width="850px" height="700" />
+
+ Similarly, you can choose timeseries chart type from the dropdown and view the chart.
```

Comment:
```suggestion
 Similarly, you can choose**Timeseries** chart type from the dropdown and view the chart.
```

Reply:
**timeseries** should be in lowercase to match the UI.